ArduCopter/GCS_Mavlink.cpp

```diff
@@ -993,6 +993,36 @@ void GCS_MAVLINK_Copter::handleMessage(mavlink_message_t* msg)
         break;
     }
 
+    case MAVLINK_MSG_ID_MANUAL_CONTROL:
+    {
+        if(msg->sysid != copter.g.sysid_my_gcs) break;                         // Only accept control from our gcs
```

Comment:
I think it is needed ! It is really bad habit even if it is only for one line, 2017 and it is still in the top10 of mistakes found atfer refactoring ^^ . and it is in the ardupilot coding style tstststststst ;-P . But I agree in this case it shouldn't be dangerous, it will be corrected when somebody make a cleanup